ISISScientificComputing/autoreduce
# Adds pytest config for running django tests and use django.urls reverse instead of hard-coded URLs

### Summary of work
Adds pytest config for running django tests - this means you can now run them with `pytest WebApp/autoreduce_webapp/selenium_tests`. You will need to install the `pytest-django` plugin. Note that tests don't pass in parallel and that will need some more work.

Uses django.urls reverse instead of hard-coded URLs. This makes us more flexible as to the actual URL names and they will be dynamically generated from what is declared in django's urls.py

### How to test your work

CI tests should pass. 

Code review.

Try running the tests locally
- `pip install pytest-django`
- `pytest WebApp/autoreduce_webapp/selenium_tests`
